localstack-core/localstack/testing/pytest/fixtures.py

```diff
@@ -904,10 +904,10 @@ def opensearch_wait_for_cluster(aws_client):
     def _wait_for_cluster(domain_name: str):
         def finished_processing():
             status = aws_client.opensearch.describe_domain(DomainName=domain_name)["DomainStatus"]
-            return status["Processing"] is False
+            return status["Processing"] is False and "Endpoint" in status
 
         assert poll_condition(
-            finished_processing, timeout=5 * 60
+            finished_processing, timeout=25 * 60, **({"interval": 10} if is_aws_cloud() else {})
```

Comment:
I suggest declaring a module level variable at the top

```
TIMEOUT = ...
INTERVAL = 10 if is_aws_cloud() else 1
```

and using it here

```suggestion
            finished_processing, timeout=TIMEOUT, interval=INTERVAL)
```

Reply:
Im fine declaring the variables, but I'd rather do it at function level. There's many poll_condition calls throughout the module, and it could be somewhat confusing to have these specific ones at module level.